spec/unit/puppet/type/rabbitmq_policy_spec.rb

```diff
@@ -195,6 +195,19 @@
     end.to raise_error(Puppet::Error, %r{Invalid delivery-limit value.*future})
   end
 
+  it 'accepts and converts the initial-cluster-size value' do
+    definition = { 'initial-cluster-size' => '3' }
+    policy[:definition] = definition
+    expect(policy[:definition]['initial-cluster-size']).to eq(3)
+  end
+
+  it 'does not accept non-numeric initial-cluster-size value' do
+    definition = { 'initial-cluster-size' => 'impressive' }
+    expect do
+      policy[:definition] = definition
+    end.to raise_error(Puppet::Error, %r{Invalid initial-cluster-size value.*future})
```

Comment:
I think a bit of copy / paste here as well. This will likely fail locally or in CI because the error should contain "impressive" vs "future". I think the `.*` is probably excessive, maybe see if `\s+` or just a space works?

Reply:
Good catch, I was too busy being funny :-( Suggested fix forthcoming